cli/bblfshctl/cmd/driver.go

```diff
@@ -0,0 +1,14 @@
+package cmd
+
+const (
+	DriverCommandDescription = "Manage drivers: install, remove and list"
+	DriverCommandHelp        = DriverCommandDescription
+)
+
+type DriverCommand struct {
+	ControlCommand
+}
+
+func (*DriverCommand) Execute([]string) error {
```

Comment:
Is this needed?

Reply:
it should fit the "interface"

Reply:
This is clearly an _abstract_ struct/class/whatever. Is still needed to implement a dummy method? If so, just ignore this comment.

Reply:
Is made by reflection, is search for a Execute method, otherwise will panic in runtime.